src/modules/transaction/components/RecentTransactions/RecentTransactions.js

```diff
@@ -63,14 +63,14 @@ const RecentTransactions = ({ className, t }) => {
         </h2>
       </BoxHeader>
       <BoxContent className={styles.content}>
-        <Table
-          data={transactions?.data || []}
-          isLoading={transactions?.isLoading}
+        <QueryTable
+          queryHook={useTransactions}
+          queryConfig={{ config: {
+            params: { limit: 5, address: host },
+            options: { enabled: !!host }
```

Comment:
I think for readability, instead of making the variable name `host`, it an be `currentAddress` or `hostAddress`

Reply:
Renamed it in the recent transactions component.